package.json

```diff
@@ -181,7 +183,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "npm run prebuild:lang && git add lang/ && lint-staged"
```

Comment:
Why is this necessary?

Reply:
I guess it's not! I thought I was adding the clones to the repo.